packages/react-nexus/src/useNexus.tsx

```diff
@@ -10,6 +10,7 @@ const warningMessage =
 
 export default function useNexus<T = any, S = any>(
   apiCall: (nexus: NexusClient) => Promise<any>,
+  inputs?: any[],
```

Comment:
As discussed, there doesn't seem to be a better way than that for now. I think it isn't that bad as it is a hook at the end of the day and the API is similar to `useEffect`. We need to document this in the README though and probably change this to
```suggestion
  inputs: any[] = [],
```